codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -307,11 +307,6 @@ let handleBlockRangeResponse = (state, ~chain, ~response: ChainWorker.blockRange
       ->Utils.unwrapResultExn
       ->updateChainFetcherCurrentBlockHeight(~currentBlockHeight)
 
-    let firstEventBlockNumber = switch parsedQueueItems[0] {
-    | Some(item) if chainFetcher.firstEventBlockNumber->Option.isNone => item.blockNumber->Some
-    | _ => chainFetcher.firstEventBlockNumber
-    }
-
```

Comment:
Previously this number was just being set on chain fetcher here. Either it existed or it didn't. Now that there can be multiple partitions even when the indexer starts up, the first batch to come back would not necessarily contain the first event.